From e1db9d5f16f0c78182dd17ab65f3974db33fc376 Mon Sep 17 00:00:00 2001
From: Youssef DALIL <youssef.dalil.pro@gmail.com>
Date: Thu, 8 Sep 2022 12:29:02 +0000
Subject: [PATCH] local.conf: adding masking configuration for gstreamer
 package

Signed-off-by: Youssef DALIL <youssef.dalil.pro@gmail.com>
---
 local.conf | 1 +
 1 file changed, 1 insertion(+)

diff --git a/local.conf b/local.conf
index d6b15c2..653c65b 100644
--- a/local.conf
+++ b/local.conf
@@ -22,3 +22,4 @@ ACCEPT_FSL_EULA = "1"
 # Switch to Debian packaging and include package-management in the image
 PACKAGE_CLASSES = "package_deb"
 EXTRA_IMAGE_FEATURES += "package-management"
+BBMASK += "gstreamer1.0 gstreamer1.0-plugins-base gstreamer1.0-plugins-good gstreamer1.0-plugins-bad"
-- 
2.17.1

