diff --git a/arista-cgt/env.sh b/arista-cgt/env.sh
index 620a021..ac2c2ee 100755
--- a/arista-cgt/env.sh
+++ b/arista-cgt/env.sh
@@ -6,7 +6,7 @@
 
 DOCKER_IMAGE_TAG="cgtimx-yocto"
 # DOCKER_IMAGE_TAG="imx-yocto"
-DOCKER_WORKDIR="/home/nick/work"
+DOCKER_WORKDIR="/home/dalilcompany/fiverr_projects/docker_arista"
 # DOCKER_WORKDIR="/opt/yocto"
 
 # Yocto
@@ -23,10 +23,14 @@ DISTRO="fsl-imx-wayland"
 IMAGES="core-image-minimal"
 # IMAGES="imx-image-core"
 
-REMOTE="https://git.congatec.com/yocto_nxp/imx6-manifest.git"
+#REMOTE="https://git.congatec.com/yocto_nxp/imx6-manifest.git"
+REMOTE="https://github.com/dalilox/cgt-custom-manifest"
 # REMOTE="https://source.codeaurora.org/external/imx/imx-manifest"
-BRANCH="cgtimx6__zeus-5.4.3-1.0.0"
+
+BRANCH="update-manifest"
+#BRANCH="cgtimx6__zeus-5.4.3-1.0.0"
 # BRANCH="imx-linux-honister"
-MANIFEST=default.xml
+
+MANIFEST=cgt-manifest-aurora-5.4.70-2.3.9.xml
 # MANIFEST=${IMX_RELEASE}".xml"
 
